ui/ui-frontend/projects/archive-search/src/app/archive/filing-holding-scheme/leaves-tree/leaves-tree.component.ts

```diff
@@ -240,6 +244,13 @@ export class LeavesTreeComponent implements OnInit, OnChanges, OnDestroy {
     return this.allunitups && this.allunitups.includes(node.id) && !this.nestedTreeControlLeaves.isExpanded(node) ? 'selected-node' : '';
   }
 
+  selectNonOprhanNodeAtTop(node: FilingHoldingSchemeNode) {
+    return this.unitId === node.id ||
+      (this.unitId !== node.id && this.allunitups && this.allunitups.includes(node.id) && !this.nestedTreeControlLeaves.isExpanded(node))
```

Comment:
Il est inutile d'ajouter la condition `this.unitId !== node` (elle découle du fait qu'on est dans un || qui suit `this.unitId === node`)
```suggestion
    return this.unitId === node.id ||
      (this.allunitups && this.allunitups.includes(node.id) && !this.nestedTreeControlLeaves.isExpanded(node))
```